Tasks/cURLUploader/curluploader.ts

```diff
@@ -41,6 +41,7 @@ async function run() {
         var curlRunner: trm.ToolRunner = tl.tool('curl');
 
         // Resolve files for the specified value or pattern
+        let uploadCount = 1;
```

Comment:
I think this along with line 125 will make it so that the warning is issued even when no files match or are found to upload, correct?  Can we not issue a warning in that case?